File: rb109/Practice/Easy 6/right_triangles.rb
# right_triangles.rb

# write a method
# input: positive integer, n
# output: triangle whose sides each have n stars

def triangle(n)
  stars = 1
  spaces = n - stars

  while stars <= n
    puts (' ' * spaces) + ('*' * stars)
    stars += 1
    spaces -= 1
  end
end

triangle(5)
triangle(9)

File: js210/js210_small_problems/easy_2/right_triangles.js
function triangle(n) {
  for (let stars = 1; stars <= n; stars += 1) {
    let line = ' '.repeat(n - stars) + '*'.repeat(stars);
    console.log(line);
  }
}

triangle(5);
triangle(9);
triangle(10);
triangle(30);